x-pack/test/performance/config.playwright.ts

```diff
@@ -32,6 +36,15 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
     },
     kbnTestServer: {
       ...functionalConfig.get('kbnTestServer'),
+      serverArgs: [
+        ...functionalConfig.get('kbnTestServer.serverArgs'),
+        '--server.uuid=5b2de169-2785-441b-ae8c-186a1936b17d',
+        '--xpack.security.encryptionKey="wuGNaIhoMpk5sO4UBxgr3NyW1sFcLgIf"', // server restarts should not invalidate active sessions
+        '--xpack.encryptedSavedObjects.encryptionKey="DkdXazszSCYexXqz4YktBGHCRkV6hyNK"',
+        '--xpack.security.session.idleTimeout=1Y',
```

Comment:
Why these values are necessary? 
Do we need them for the scalability testing?

Reply:
we had issues with sharing auth state, it's not necessary anymore I'm deleting it 👍🏽